src/main/java/com/epam/lowcost/DAO/FlightDAO.java

```diff
@@ -11,5 +12,6 @@
     Flight addNewFlight(Flight flight);
     Flight deleteFlight(Long id);
     Flight updateFlight(Flight flight);
+    List <Flight> getByFromToDate (String departureAirport, String arrivalAirport, LocalDateTime departureDate,LocalDateTime arrivalDate);
```

Comment:
Is this filter really necessary ?

Reply:
yes, we need it when user wants to find flight exactly for his/her demand

Reply:
What do you do in real life when you choose flight? You choose airports(from/to), and range of date only of departure. You never choose range between departure time and arrival time, don't you ?

Reply:
yes, this is exactly what happens in the code, arrivalDate is only the name of variable

Reply:
corrected